test/dhtest.c

```diff
@@ -485,14 +491,19 @@ static const rfc5114_td rfctd[] = {
 static int run_rfc5114_tests(void)
 {
     int i;
+    DH *dhA = NULL, *dhB = NULL;
+    unsigned char *Z1 = NULL, *Z2 = NULL;
+ 
     for (i = 0; i < (int)OSSL_NELEM(rfctd); i++) {
-        DH *dhA, *dhB;
-        unsigned char *Z1 = NULL, *Z2 = NULL;
+        dhA = NULL;
+        dhB = NULL;
+        Z1 = NULL;
+        Z2 = NULL;
         const rfc5114_td *td = rfctd + i;
```

Comment:
In C, you cannot mix declarations and statements like this. Move the declaration of `rfc5114_td` up and you will see your Travis trouble go away.